src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx

```diff
@@ -0,0 +1,260 @@
+
+### Create a service level for each serverless application
+Create an error rate service level for AWS Lambda function integrated with our [serverless monitoring](/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/monitoring-aws-lambda-serverless-monitoring):
+* For Step 1, select `Lambda function` as the entity type
+* For Step 2, select `AWSLambdaInvoation` for valid events and `AwsLambdaInvocationError` for bad responses
+
+Currently, service levels only support error rates for AWS Lambda functions using [New Relic serverless monitoring for AWS Lambda](/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/monitoring-aws-lambda-serverless-monitoring). You can capture the error rate outside of service levels using the following query:
+```
+SELECT sum(provider.errors.Sum)/sum(provider.invocations.Sum)*100 FROM ServerlessSample
+```
+
+### Create a service level for each OpenTelemetry application
+
+* For Step 1, select `Service - OpenTelemetry`.
+* For Step 2's valid events, use the `Span` entity type for good event types. Add the following to the WHERE clause:
+  ```(span.kind LIKE 'server' OR span.kind LIKE 'consumer' OR kind LIKE 'server' OR kind LIKE 'consumer')``` 
+* For Step 2's invalid events, use the `Span` entity type and `Repeat WHERE clause` option. Add the following to the WHERE clause to detect bad responses:
+  ```otel.status_code = 'ERROR'```
+
+## Improvement process [#improvement-process]
+* [Identify inconsequential errors](#create-workload)
+* [Remove inconsequential errors from your error rate](#create-service-level)
+* [Set up error rate alerts](#error-rate-alerts)
+* [Establish an error-hero roster](#error-hero)
+* [Triage errors using errors inbox](#errors-inbox)
+* [Link errors to JIRA](#jira)
+* [Link errors to Slack](#slack)
+* [Use codestream](#codestream)
+
+### Identify inconsequential errors [#inconsequential-errors]
+Explore your errors however you are most comfortable. You can do this using: 
+* Out-of-the-box views for APM, mobile monitoring, JavaScript errors, serverless monitoring, and OpenTelemetry
+* Errors inbox filtered for your workload
+* NRQL data types such as `TransactionError`, `JavaScriptError`, `MobileRequestError`, `AwsLambdaInvocationError`, `Span`
+
+### Remove inconsequential errors from your error rate [#remove-inconsequential-errors]
+
+You can remove errors that don't matter in one of two ways:
+* Stop them from being ingested using [configuration](/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected/#ignore) (APM only) or using [drop rules](/docs/data-apis/manage-data/drop-data-using-nerdgraph). This approach only works for errors that you are certain don't need capturing. The added advantage of this approach is reduction in ingest for noisy errors.
```

Comment:
unsure if 'this approach only works...' was referring to 'dropping data' or to both dropping data and AMP config.